src/gui/MainWindow.cpp

```diff
@@ -77,7 +78,7 @@ class HttpPlugin: public ISettingsPage
 
     QString name() override
     {
-        return QObject::tr("Browser Integration");
+        return QObject::tr("Browser Integration (old)");
```

Comment:
Recommend "Legacy Browser Integration", changing the first word makes it stand out more. We should also change the icon, I recommend a NEW icon for the new plugin, maybe [something like this](https://visualpharm.com/assets/591/Open%20In%20Browser-595b40b75ba036ed117da0be.svg).

Reply:
I wanted to name it like that, but it's too long and I know it's even longer in other languages. Depending on font size and DPI settings, stuff may get cut off. Which in the other hand, is another argument in favor of putting it first, but "old browser integration" sounded a little too naive to me.
 Windows is special with its tiny font size. You could fit a novel there.

Reply:
For the icon, I'd like to have a different one, too, but we don't really have one. The one you linked looks completely out of place. A new icon needs to be consistent with the rest of the Oxygen icons. Or we need a fitting new logo for KeepasXC-Browser which is not just the KeePassXC logo.

Reply:
I meant "like" it, not exactly that icon. Here is another idea:

(Add the KeePassXC Logo in the background on the computer monitor) http://www.iconarchive.com/show/oxygen-icons-by-oxygen-icons.org/Apps-krfb-icon.html